src/scripts/main.js

```diff
@@ -354,7 +354,29 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('table');
 
-// write your code here
+for (let i = 0; i < people.length; i++) {
+  const tr = document.createElement('tr');
+  const th1 = document.createElement('th');
+  const th2 = document.createElement('th');
+  const th3 = document.createElement('th');
+  const th4 = document.createElement('th');
+  const th5 = document.createElement('th');
+  const th6 = document.createElement('th');
+
+  th1.textContent = people[i].name;
+  th2.textContent = people[i].sex;
+  th3.textContent = people[i].born;
+  th4.textContent = people[i].died;
+  th5.textContent = people[i].died - people[0].born;
```

Comment:
The calculation here is incorrect. You are subtracting the birth year of the first person in the array from the death year of the current person. This should be corrected to calculate the age of each person by subtracting their birth year from their death year: `people[i].died - people[i].born`.